src/utils/lesson-metadata.ts

```diff
@@ -51,7 +52,7 @@ export const mergeLessonMetadata = (
     ? primaryInstructor
     : secondaryInstructor
 
-  const rest = {...secondaryRest, ...primaryRest}
+  const rest = compactedMerge(secondaryRest, primaryRest)
```

Comment:
compacted merge so that we don't have something like `duration: null` overriding an actual `duration: 123` value.